application/healthcare-services-data-manager/common/common_functions.py

```diff
@@ -0,0 +1,119 @@
+import os
```

Comment:
I think this is a temporary file created by a local run of the terraform. Maybe we could look at a gitignore to prevent these creeping in ?

Reply:
removed and updated gitignore